### ISSUE TITLE
Command and CanExecute not working on Label

### ISSUE THREAD

**Issue by highrobot** [mrxten/XamEffects#22]
I have tried many things but it seems to be a bug that I cannot solve alone. Everything works fine with my implementation except when I try to prevent the execution with a property set to false. The Observation is never taken into consideration and execute the command method immediately. Hereby is the code :

```
<Label  Text="Test XamEffects"  
xamEffects:TouchEffect.Color="{StaticResource XamEffectsColor}" 
xamEffects:Commands.Tap="{Binding ShowDetails}"/>
```
ViewModel: 
`ShowDetails= new DelegateCommand<object>(ExecuteShowCertificateDetails).ObservesCanExecute(() => CanNavigate);`

Here I am using Prism but I have the same result if I switch to Xamarin default Command and canExecute method. Note that it is working as expect if I use Tapgesturerecognizer.

**Comment by mrxten** [comment]
Yes, you right. Command calling even if Command.CanExecute == false

**Comment by mrxten** [comment]
Fixed in 1.5.4
